README.md

```diff
@@ -1,6 +1,6 @@
 ##SuiteCRM 7.6.3
 
-[![Build Status](https://travis-ci.org/salesagility/SuiteCRM.svg?branch=hotfix)](https://travis-ci.org/salesagility/SuiteCRM)
```

Comment:
Can not merge due to this.  You will need to update your hotfix branch and re-create this issue branch to only commit files that you want i.e. not this one.